File: TravellerCreator/ClientApp/src/app/sector-creator/models/star-frontiers/star-frontiers-sector.ts
import {ISector} from "../interfaces/sector";
import {StarFrontiersSubsector} from "./star-frontiers-subsector";
import {SectorType} from "../enums/sector-type";

export class StarFrontiersSector implements ISector {
  sectorType: SectorType = SectorType.StarFrontiers;
  subsectors: StarFrontiersSubsector[];
}

File: TravellerCreatorModels/StarFrontiers/StarFrontiersSector.cs
using TravellerCreatorModels.Enums;
using TravellerCreatorModels.Interfaces;

namespace TravellerCreatorModels.StarFrontiers;

public class StarFrontiersSector : ISector
{
    public SectorType SectorType => SectorType.StarFrontiers;
    public List<ISubsector> Subsectors { get; set; } = new();
}